include/directx/d3dx12.h

```diff
@@ -4072,6 +4072,8 @@ if (FAILED(m_hStatus = m_pDevice->CheckFeatureSupport(FEATURE, &MEMBER, sizeof(M
     return m_hStatus; \
 }
 
+#define INITIALIZE_FAILED(FEATURE, MEMBER) FAILED(m_hStatus = m_pDevice->CheckFeatureSupport(FEATURE, &MEMBER, sizeof(MEMBER)))

```

Comment:
I'm pretty sure you don't want to assign `m_hStatus` as part of this macro. Otherwise, the overall success/failure of the entire initialization process will just boil down to the last feature that was checked.

Reply:
Oh! My bad. I didn't want to do this assignment! Thanks for pointing out.